modules/os2forms_forloeb/src/MaestroHelper.php

```diff
@@ -0,0 +1,626 @@
+<?php
+
+namespace Drupal\os2forms_forloeb;
+
+use DigitalPost\MeMo\Action;
+use DigitalPost\MeMo\EntryPoint;
+use Drupal\advancedqueue\Entity\QueueInterface;
+use Drupal\advancedqueue\Job;
+use Drupal\advancedqueue\JobResult;
+use Drupal\Component\Render\MarkupInterface;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Config\ImmutableConfig;
+use Drupal\Core\Entity\EntityStorageInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Extension\ModuleHandlerInterface;
+use Drupal\Core\Language\LanguageManagerInterface;
+use Drupal\Core\Logger\LoggerChannelInterface;
+use Drupal\Core\Mail\MailManagerInterface;
+use Drupal\Core\Render\Markup;
+use Drupal\Core\Url;
+use Drupal\entity_print\Plugin\EntityPrintPluginManagerInterface;
+use Drupal\maestro\Engine\MaestroEngine;
+use Drupal\maestro\Utility\TaskHandler;
+use Drupal\os2forms_digital_post\Helper\DigitalPostHelper;
+use Drupal\os2forms_digital_post\Model\Document;
+use Drupal\os2forms_forloeb\Exception\RuntimeException;
+use Drupal\os2forms_forloeb\Form\SettingsForm;
+use Drupal\os2forms_forloeb\Plugin\AdvancedQueue\JobType\SendMeastroNotification;
+use Drupal\os2forms_forloeb\Plugin\EngineTasks\MaestroWebformInheritTask;
+use Drupal\os2forms_forloeb\Plugin\WebformHandler\MaestroNotificationHandler;
+use Drupal\webform\WebformSubmissionInterface;
+use Drupal\webform\WebformSubmissionStorageInterface;
+use Drupal\webform\WebformThemeManagerInterface;
+use Drupal\webform\WebformTokenManagerInterface;
+use ItkDev\Serviceplatformen\Service\SF1601\SF1601;
+use Psr\Log\LoggerInterface;
+use Psr\Log\LoggerTrait;
+
+/**
+ * Maestro helper.
+ */
+class MaestroHelper implements LoggerInterface {
+  use LoggerTrait;
+
+  public const OS2FORMS_FORLOEB_NOTIFICATION_ASSIGNMENT = 'assignment';
+  public const OS2FORMS_FORLOEB_NOTIFICATION_REMINDER = 'reminder';
+  public const OS2FORMS_FORLOEB_NOTIFICATION_ESCALATION = 'escalation';
+
+  /**
+   * The config.
+   *
+   * @var \Drupal\Core\Config\ImmutableConfig
+   */
+  readonly private ImmutableConfig $config;
```

Comment:
although i couldn't find any speicitail, but all the examples had readonly modifier positioned after the visiblity modifier.

e.g. `private readonly ImmutableConfg $config;`

you can leave it like that here is you think it's not important

Reply:
I don't know where that order comes from. Fixed in https://github.com/OS2Forms/os2forms/pull/56/commits/859355af527a3ce6c44feb262aec55819d7e61d8.